docs/commands.rst

```diff
@@ -306,6 +306,51 @@ and to rollback use `<http://localhost:8000/rollback>`__.
         or other another dynamic option, set ``$_SERVER['PHINX_DBNAME']`` before
         running commands. Available options are documented in the Configuration page.
 
+Wrapping Phinx in another Symfony Console Application
+-----------------------------------------------------
+
+Phinx can be wrapped and run as part of a separate Symfony console application. This
+may be desirable to present a unified interface to the user for all aspects of your
+application, or because you wish to run multiple Phinx commands. While you could
+run the commands through ``exec`` or use the above ``Phinx\Wrapper\TextWrapper``,
+though this makes it hard to deal with the return code and output in a similar fashion
+as your application.
+
+Luckily, Symfony makes doing this sort of "meta" command straight-forward:
```

Comment:
Does the final : needs to be :: ?
I am not too familiar with this rst stuff, but I thought this is needed to quote this char.

Reply:
You use the `::` to make something a [Literal Block](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks), which is handled by the `::` on line 321 (`.. code-block:: php`). Here I'm just using the `:` as a `:` to signify the connected content is coming on the next bit.